automated-e2e-test/README.md

```diff
@@ -0,0 +1,171 @@
+# Welcome to the SODA Automated E2E Test Suite
+Automated E2E tests for SODA Project APIs using Postman collections, global variables and the [Newman](https://www.npmjs.com/package/newman) collection runner.  
+The [newman-reporter-htmlextra](https://www.npmjs.com/package/newman-reporter-htmlextra) is a HTML reporter that has been extended to include the separation of the iteration runs.  
+This reporter comes with a dashboard style summary landing page and a set of different tabs which contain the detailed request information. There are also a few optional [configuration flags available](https://www.npmjs.com/package/newman-reporter-htmlextra#user-content-usage), to tailor the final report in a number of different ways.
+
+
+## Pre-requisites
+---  
+- To run Newman, ensure that you have Node.js >= v10. [Install Node.js via package manager](https://nodejs.org/en/download/package-manager/).  
+You can install different versions of NodeJS and maintain different versions using `nvm - Node Version Manager`. Instructions on how to install `nvm` can be found [here](https://github.com/nvm-sh/nvm)  
+
+- A working SODA installation.
+- For Multicloud testing, generate an AK/SK using the SODA Dashboard. Currently the AK/SK can be generated only using the SODA Dashboard. 
+
+## Installation
+--- 
+1. Clone the repository  
+
+    ```
+    git clone https://github.com/sodafoundation/qa/.git
+    ```
+
+2. To Install `newman` and the reporter `newman-reporter-htmlextra`  run the following:
+
+    ```
+    cd automated-e2e-test
+    npm install
+    ```
+
+
+
+## Configuration
+
+- Update the file `./env_vars/SODA.postman_globals.json`
```

Comment:
> placing single storage sync just after sync all might cause TC failure, when the first sync is going on, second sync would return 400. either we introduce some delay, or keep some gap in between syncs

Yes. I faced the same issue. There is a delay of 2000ms between calls, but I think it is not enough. We can explore how to resolve this.

> File name is SODA_Globals.postman_globals.json

Right. I think the readme was not updated after I made the changes, I will make the change. But right now since the app takes care of the config you don't have to make this change unless you are customizing the file name.